frontend/src/metabase/metabot/actions.ts

```diff
@@ -87,6 +91,11 @@ export const RUN_QUESTION_QUERY_REJECTED =
 export const runQuestionQuery =
   () => async (dispatch: Dispatch, getState: GetState) => {
     try {
+      const question = getQuestion(getState());
+      if (question) {
+        await dispatch(updateQuestion(getRawTableQuestion(question)));
```

Comment:
We can't use the same display/settings from the prompt question, so we have to set a raw question when running the edited query manually